src/test/datascience/notebook/helper.ts

```diff
@@ -293,30 +296,54 @@ export async function trustAllNotebooks() {
     }
     (<any>dsSettings).alwaysTrustNotebooks = true;
 }
-export async function startJupyter(closeInitialEditor: boolean) {
+
+export async function startRemoteJupyterServer() {
```

Comment:
This seems like it's named incorrectly. It doesn't always start a remote server.

Maybe it would be better to call it 'connectToServer' or 'startServer'?

Reply:
Not connecting either
We have one called `startJupyter`, will rename that to `prewarmNotebooks` and rename this to `startJupyterServer`

Reply:
Fixed